### DISCUSSION TITLE
[2021.09.08] 다리를 지나는 트럭

### DISCUSSION THREAD

**Discussion by chaeyeon4u** [pyunji/Hyundai-ITnE-Team_QuadCore#119] [Algorithm]
[다리를 지나는 트럭](https://programmers.co.kr/learn/courses/30/lessons/42583)

**Comment by hyk339** [comment]
[다리를 지나는 트럭 solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/blob/master/khy/programmers/PR_42583.java)

현지님이 알려주신 방법대로 보지 않고 구현해보았습니다.

**Reply by pyunji** [reply]
멋져부러요

**Comment by pyunji** [comment]
[다리를 지나는 트럭 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/PR/PR_42587.java)

**Comment by chaeyeon4u** [comment]
[다리를 지나는 트럭 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/Java/PR_42583.java)

**Comment by leechang4** [comment]
[다리를 지나는 트럭 solved](https://github.com/leechang4/Hyundai-ITnE-Team_QuadCore/blob/master/lch/Programmers/PR_42583.java)
